src/test/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/BeslutteVedtakStegTest.kt

```diff
@@ -174,18 +181,45 @@ internal class BeslutteVedtakStegTest {
 
     @Test
     internal fun `skal ikke ha beslutter ved avslag og mindre inntektsendringer`() {
-        every { vedtakService.hentVedtak(any()) } returns vedtak(behandlingId, resultatType = ResultatType.AVSLÅ).copy(avslåÅrsak = AvslagÅrsak.MINDRE_INNTEKTSENDRINGER)
-        every { vedtaksbrevService.lagEndeligBeslutterbrev(any(), vedtakErUtenBeslutter) } returns Fil("123".toByteArray())
+        every { vedtakService.hentVedtak(any()) } returns vedtak(behandlingId, resultatType = ResultatType.AVSLÅ).copy(
+            avslåÅrsak = AvslagÅrsak.MINDRE_INNTEKTSENDRINGER
+        )
+        every {
+            vedtaksbrevService.lagEndeligBeslutterbrev(
+                any(),
+                vedtakErUtenBeslutter
+            )
+        } returns Fil("123".toByteArray())
```

Comment:
Synes ktlint gir litt rare forslag ibland. Her kan man endre om litt tenker jeg, eks
```suggestion
        every { vedtakService.hentVedtak(any()) } returns 
            vedtak(behandlingId, resultatType = ResultatType.AVSLÅ)
                .copy(avslåÅrsak = AvslagÅrsak.MINDRE_INNTEKTSENDRINGER)
        every {
            vedtaksbrevService.lagEndeligBeslutterbrev(any(), vedtakErUtenBeslutter)
        } returns Fil("123".toByteArray())
```

Reply:
Enig, takk :)